server/prisma-rs/migration-engine/connectors/sql-migration-connector/src/database_inspector/mod.rs

```diff
@@ -26,7 +26,7 @@ pub trait DatabaseInspector: Send + Sync + 'static {
     fn introspect(&self, schema: &String) -> DatabaseSchema;
 }
 
-impl DatabaseInspector {
```

Comment:
You could maybe move all of these methods directly to the trait.

Reply:
@pimeys Could we do that in another PR? I tried, but it leads to non-trivial build failures.

Reply:
Can you show me what kind of failures? You're doing God's work here btw :)

Reply:
@pimeys OK, I'll try to reproduce

Reply:
@pimeys After discussing with Marcus, we'll merge this now and handle the trait in a follow-up PR. I'll get back to you later on this, OK? Got to handle another issue now.

Reply:
See https://github.com/prisma/prisma/pull/4836.